release notes/v0.35.0.md

```diff
@@ -0,0 +1,97 @@
+
+### Other notable changes
+
+- k6's GRPC was extended with a support for server reflection which allows one to use GRPC even without `proto` file at hand. You can read more about the protocol [here](https://github.com/grpc/grpc/blob/master/doc/server-reflection.md) ([#2160](https://github.com/grafana/k6/pull/2160)). Thanks, @joshcarp!
+- Support for cookie jars in `k6/ws` ([#2193](https://github.com/grafana/k6/pull/2193)).
+- Forbid `metric.Add` to let `NaN` values through ([#1876](https://github.com/grafana/k6/pull/1876)); instead, print out nice warnings or throw an exception if `--throw` is enabled. These warnings will not become too large ([#2219](https://github.com/grafana/k6/pull/2219)).
+- Support for compression in websockets ([#2162](https://github.com/grafana/k6/pull/2162)). Thanks, @cooliscool!
+- Switch to camel case for CLI options to the outputs ([#2150](https://github.com/grafana/k6/pull/2150)). Thanks, @JosephWoodward!
+
+## Breaking changes
+
+- Addition of common metrics registry no longer allows defining custom metrics with the same name as one of the builtin metrics, e.g. `new Counter("http_req_duration")` will abort. Similarly, an attempt to redefine a metric with the same name but with different type will error out. Builtin metrics may no longer be referenced as global objects in xk6 extensions either ([#2071](https://github.com/grafana/k6/pull/2071)).
+- Fix inconsistency in environment variables' names: use `K6_NO_SETUP` and `K6_NO_TEARDOWN` options instead of `NO_SETUP` and `NO_TEARDOWN` ([#2140](https://github.com/grafana/k6/pull/2140)).
+
+## Bugs fixed!
+
+- Fix of a misleading sorting of custom submetrics ([#2198](https://github.com/grafana/k6/pull/2198)). Thanks, @knittl!
+- Fix for extensions depending on `afero.FS`: implement a newer version of the `afero.FS` interface for internal filesystems so that extension depending on that or newer version can be built ([#2216](https://github.com/grafana/k6/pull/2216)).
+- Fix for websockets: websockets now use the global `User-Agent` setting ([#2151](https://github.com/grafana/k6/pull/2151)). Thanks, @cooliscool!
+- Fixes for tests, Github actions, and Loki integration ([#2205](https://github.com/grafana/k6/pull/2205), [#2153](https://github.com/grafana/k6/pull/2153), [#2220](https://github.com/grafana/k6/pull/2220)).
+
+## Maintenance
+
+- Update a whole bunch of dependencies ([#2159](https://github.com/grafana/k6/pull/2159), [#2170](https://github.com/grafana/k6/pull/2170), [#2165](https://github.com/grafana/k6/pull/2165)).
+- Switch to Go 1.17 ([#2156](https://github.com/grafana/k6/pull/2156)). Thanks, @b5710546232!
+- Get rid of `mapstructure` ([#2223](https://github.com/grafana/k6/pull/2223)).
+- Much neater error message on `nil` response body ([#2195](https://github.com/grafana/k6/pull/2195)). Thanks, @daniel-shuy!
```

Comment:
This should probably be moved to the `Other notable changes and improvements` section, it has user-facing changes :man_shrugging:

Reply:
Not sure about "notable" here but if GRPC is important enough to be moved out into a separate part, that section with "Other notable changes" no longer seems as large :sweat_smile:

Reply:
hmm yeah... this section used to be called `Enhancements and UX improvements` in old release notes, so maybe `Other changes and UX improvements`? :man_shrugging:

Reply:
:+1: yes, that seems more fitting now

Reply:
Update is up.